### PR TITLE
fix(serverless): gracefully fail with 400 for null body

### PR DESCRIPTION
**Pull request by decompil3d**
It is apparently possible to get a null `event.body` from API Gateway. Right now, this leads to a TypeError at [this `console.log` statement](https://github.com/probot/serverless-lambda/blob/master/index.js#L61) due to the access of `event.body.action`. This PR adds logic to gracefully fail early with a 400 error.

### CODE REVIEW

**Comment by decompil3d** [comment]
@JasonEtco any chance you can help get this looked at? Thanks!

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "url": "git+https://github.com/probot/serverless-lambda.git"
   },
   "devDependencies": {
+    "@types/jest": "^26.0.0",
```
Comment (JasonEtco) [inline package.json:14]: Can you undo this change, and the change to the `package-lock.json`? We don't need `@types/jest`

**Review by gr2m — APPROVED** [review]
cheers!
